src/plugins/obj/obj_engine_registry.cpp

```
@@ -0,0 +1,54 @@
+objAccelEngineRegistry::instance() {
+    static objAccelEngineRegistry registry;
+    return registry;
+}
+
+void
+objAccelEngineRegistry::add(const std::string &type, objAccelEngineEntry entry) {
+    entries_[type] = std::move(entry);
+}
+
+bool
+objAccelEngineRegistry::has(const std::string &type) const {
```

Cppcheck (2.20.0)

[style] The function 'has' is never used.

(unusedFunction)